features/support/world.js

```diff
@@ -10,13 +10,11 @@ require('../../bin/www'); // This starts the web server, and ensures it is only
 
 function World() {
   this.expect = chai.expect;
-
+  Zombie.site = 'http://localhost:3000';
```

Comment:
I've only found a static property https://github.com/assaf/zombie/blob/master/src/index.js#L1320 It will probably have to stay like this.